server/src/main/java/io/druid/server/coordinator/DruidCoordinator.java

```diff
@@ -231,8 +231,48 @@ public boolean isLeader()
     return retVal;
   }
 
+  // updated by druid.coordinator.period
   public CountingMap<String> getSegmentAvailability()
   {
+    return getSegmentStats(
+        new Function<DataSegment, Integer>()
+        {
+          @Override
+          public Integer apply(DataSegment input)
+          {
+            return segmentReplicantLookup.getTotalReplicants(input.getIdentifier()) == 0 ? 1 : 0;
+          }
+        }
+    );
+  }
+
+  // updated by druid.coordinator.period
+  public CountingMap<String> getLoadPendingSegments()
```

Comment:
It's expected to be used but I've found it's updated by commented period, which seemed not appropriate for small polling like testcase. Should we remove this?